docs/src/components/FeedbackCallout.js

```diff
@@ -0,0 +1,72 @@
+// @flow strict
+import { Box, Button, Callout, Flex, Layer, Modal } from 'gestalt';
+import React, { type Node } from 'react';
+
+const FeedbackCallout = (): Node => {
+  const [showCallout, setShowCallout] = React.useState(true);
+  const [showModal, setShowModal] = React.useState(false);
+
+  return (
+    <Box marginLeft={-1} marginRight={-1}>
+      {showCallout && (
+        <Callout
+          type="info"
+          iconAccessibilityLabel="Info icon"
+          title="We'd love your feedback!"
+          message="The Gestalt team has been working hard to extend our documentation to include design best practices, as well as create a standardized documentation format. Let us know what you think so far by filling out our survey."
+          primaryAction={{
+            label: 'Give feedback',
+            onClick: () => {
+              setShowModal(!showModal);
+            },
+          }}
+          dismissButton={{
+            accessibilityLabel: 'Dismiss banner',
+            onDismiss: () => {
+              setShowCallout(!showCallout);
+            },
+          }}
+        />
+      )}
+      {showModal && (
+        <Layer>
+          <Modal
```

Comment:
we are losing shadows

![image](https://user-images.githubusercontent.com/10593890/109579711-6fefd000-7ac7-11eb-8e8d-3f3cfa457828.png)

Reply:
Not an expert but remember @christianvuerings mentioning iframes (related to [security](https://www.ostraining.com/blog/webdesign/against-using-iframes/#:~:text=Iframes%20Bring%20Security%20Risks,phishing%20your%20users'%20personal%20data.&text=A%20malicious%20user%20can%20hijack%20your%20users'%20keystrokes.)?) issues and suggesting using an external link

Reply:
We ended up switching to just a link, so no iframe :D